Travel Scheduler.xcodeproj/project.pbxproj

```diff
@@ -540,7 +551,7 @@
 				C027E49C22EF63D700583555 /* Gotham-Black.otf in Resources */,
 				C027E49E22F0BD9100583555 /* Gotham-Font in Resources */,
 				C027E49422EF63D700583555 /* Gotham-Bold.otf in Resources */,
-				7194AA1322EA01B600949669 /* random_50fps.gif in Resources */,
+				7194AA1322EA01B600949669 /* random_50fpsgif.gif in Resources */,
```

Comment:
I think we can change it to another name lol

Reply:
Yeah I agree. Random_Lhama.gif